perception/elevation_map_loader/config/elevation_map_parameters.yaml

```diff
@@ -1,30 +0,0 @@
-pcl_grid_map_extraction:
-  num_processing_threads: 12
```

Comment:
Why did you remove `transform` and `rotation` parameters?

Reply:
Just added other parameters, please check out now